.github/workflows/periodic.yaml

```diff
@@ -23,11 +23,9 @@ jobs:
     steps:
     - name: Clone scripts
       uses: actions/checkout@v2
-    - name: Install NPM dependencies
-      run: npm install --prefix .github/actions/pick
     - name: Pick last successful commit
       id: pick
-      uses: ./.github/actions/pick
+      uses: clangd/actions/.github/actions/pick@v0.1.0
```

Comment:
.we could consider `@master` to reduce toil since we're fairly tightly coupled.
Up to you, tag number is more precise (though we don't actually have reproducibility here)

Reply:
i've suggested tag numbers to give us some slack in case we do some incompatible changes, e.g. rename/add/drop parameters.